components/org.wso2.carbon.identity.api.user.authorized.apps/org.wso2.carbon.identity.rest.api.user.authorized.apps.v2/src/gen/java/org/wso2/carbon/identity/rest/api/user/authorized/apps/v2/ApplicationsApi.java

```diff
@@ -0,0 +1,64 @@
+/*
+* Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package org.wso2.carbon.identity.rest.api.user.authorized.apps.v2;
+
+import io.swagger.annotations.*;
```

Comment:
Wildcard imports

Reply:
Addressed in [01e1307](https://github.com/wso2/identity-api-user/pull/129/commits/01e13074b77a93e08fd8c7d276b04f1525b274c5)

Reply:
for auto generated classes, it is ok to have wildcard imports